exchange/exchangetest/multi-bids-different-ortb-versions.json

```diff
@@ -0,0 +1,279 @@
+                    "id": "some-request-id",
+                    "site": {
+                        "page": "test.somepage.com"
+                    },
+                    "user": {
+                        "geo": {},
+                        "consent": "AAAA",
+                        "eids": [
+                            {
+                                "source": "source",
+                                "uids": [
+                                    {
+                                        "id": "1",
+                                        "atype": 1,
+                                        "ext": {}
+                                    },
+                                    {
+                                        "id": "1",
+                                        "atype": 1,
+                                        "ext": {}
+                                    }
+                                ],
+                                "ext": {}
+                            }
+                        ],
+                        "ext": {}
+                    },
+                    "imp": [
+                        {
+                            "rwdd": 1,
+                            "id": "some-imp-id",
+                            "banner": {
+                                "format": [
+                                    {
+                                        "w": 600,
+                                        "h": 500
+                                    },
+                                    {
+                                        "w": 300,
+                                        "h": 600
+                                    }
+                                ]
+                            },
+                            "ext": {
+                                "bidder": {
+                                    "placementId": 1
+                                }
+                            }
+                        }
+                    ],
+                    "source": {
+                        "fd": 1,
+                        "tid": "abc123",
+                        "pchain": "tag_placement",
+                        "schain": {
+                            "complete": 1,
+                            "nodes": [
+                                {
+                                    "asi": "asi",
+                                    "sid": "sid",
+                                    "rid": "rid",
+                                    "ext": {}
+                                }
+                            ],
+                            "ver": "ver",
+                            "ext": {}
+                        },
+                        "ext": {}
+                    },
+                    "regs": {
+                        "coppa": 1,
+                        "ext": {
+                            "gdpr": 1,
+                            "us_privacy": "1YYY"
+                        }
+                    }
```

Comment:
Since appnexus is declared as a 2.6 bidder, this should be:
```
"regs": {
    "coppa": 1,
    "gdpr": 1,
    "us_privacy": "1YYY"
}
```
I expect this to cause the test to fail as we have identified a shared memory issue with respect to `regs`.

Reply:
Fixed input request and outgoing request for appnexus, good point!